src/components/SecondaryPanes/Breakpoints.css

```diff
@@ -53,18 +53,26 @@ html .breakpoints-list .breakpoint.paused {
   border-color: var(--breakpoint-active-color-hover);
 }
 
-.breakpoints-list .breakpoint-checkbox {
-  margin-inline-start: 0;
-  vertical-align: -2px;
-}
-
 .breakpoints-list .breakpoint-label {
+  max-width: calc(100% - 36px);
```

Comment:
lets move the 36px into a var, having it as a hard-coded value is a bit risky

Reply:
@codehag created a shared css var for breakpoint and expression